### PR TITLE
Release 3.0.0

### PR DESCRIPTION
**Pull request by alexjfisher**

### CODE REVIEW

**Comment by alexjfisher** [comment]
The changelog is a bit messy.  If anyone wants to massage the various linked issues and PRs, I can then regenerate it.

You can improve the english in issue and PR titles, or apply labels such as `invalid` or `question` to things you think could be excluded completely.

File: HISTORY.md [hunk 1]
```diff
@@ -1,36 +1,27 @@
-Change log for hiera-eyaml
-==========================
-
-2.1.0
------
+## v2.1.0 (2016-03-02)
```
Comment (ghoneycutt) [inline HISTORY.md]: Suggest removing this file as it is a duplicate of CHANGELOG.md

Comment (alexjfisher) [reply HISTORY.md]: The old CHANGES.md becomes HISTORY.md which github changelog generator then uses when constructing the complete CHANGELOG.md.  It automatically picks up `2.1.0` as the `since_tag`.  See https://github.com/github-changelog-generator/github-changelog-generator#migrating-from-a-manual-changelog

The manually curated CHANGES.md is probably better than what gcg would spew out if I let it look through the project's complete history.

File: HISTORY.md [hunk 2]
```diff
@@ -1,36 +1,27 @@
...
 
  - (#187) - Change the way third party highline library is imported to avoid memory leak when running under puppet server (@petems)
  - (#181) - Improve test suite to run against a variety of puppet versions (@peculater)
 
-2.0.8
------
+##v2.0.8 (2015-04-15)
```
Comment (ekohl) [inline HISTORY.md]: This may be invalid markdown. Most parsers require a space after `##`.

Comment (alexjfisher) [reply HISTORY.md]: Good spot.  Fixed.

**Pull request merged**